### DISCUSSION TITLE
Need to be on while charging?

### DISCUSSION THREAD

**Discussion by Pandagineer** [bitcraze/discussions#329] [Q&A] [answered]
Quick question: when plugged in for charging, does the Crazyflie need to be turned on? It was most recently at around 3.7V. I have it plugged in right now, turned off, and I don't see any LEDs to indicate charging. Thanks!

**Comment by krichardsson** [comment, marked as answer]
It charges also if not turned on, but only with 100mA.
If turned on it charges with around 400mA

**Reply by Pandagineer** [reply]
Thank you.
